src/main/java/org/stellar/sdk/Transaction.java

```diff
@@ -51,6 +52,14 @@ public void sign(KeyPair signer) {
     mSignatures.add(signer.signDecorated(txHash));
   }
 
+  public static void setDefaultBaseFee(int baseFee) {
```

Comment:
I might rename this to `setDefaultFee`, since the [base fee](https://www.stellar.org/developers/guides/concepts/fees.html) is fixed at 100 stroops (at least for now). I assume this is meant to set the fee that will be used for transactions submitted in this session, and could easily be set to something other than 100 stroops even if the network base fee doesn't change.

Reply:
yes, good point, will rename. thanks.

Reply:
will rename to `setDefaultOperationFee` to keep it more descriptive (bartek's comment below)